plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraTableHandle.java

```diff
@@ -77,7 +78,7 @@ public String getClusteringKeyPredicates()
 
     public SchemaTableName getSchemaTableName()
     {
-        return new SchemaTableName(schemaName, tableName);
+        return new SchemaTableName(schemaName.toLowerCase(ENGLISH), tableName.toLowerCase(ENGLISH));
```

Comment:
Anyways it would break for all connectors...Unless we use a different spi object - which adds further complexity. We have ensured that things doesn't before introducing this change..